content/api/entity-reference/volunteer/index.html

```diff
@@ -134,32 +131,32 @@
             <tr>
                 <td>first_year_volunteered</td>
                 <td>integer</td>
-                <td>The first year volunteered.</td>
+                <td>This is a computed field based on the volunteer's first year volunteered.</td>
             </tr>
             <tr>
                 <td>total_hours</td>
                 <td>decimal</td>
-                <td>The total hours volunteered.</td>
+                <td>This is a computed field based on the volunteer's total hours volunteered.</td>
             </tr>
             <tr>
                 <td>total_value</td>
                 <td><a href="/constituent-entity-reference#Currency">currency</a></td>
-                <td>Estimated monetary value of the work that the volunteer completed.</td>
+                <td>This is a computed field based on the estimated monetary value of the work that the volunteer has completed.</td>
             </tr>
             <tr>
                 <td>total_years_volunteered</td>
                 <td>integer</td>
-                <td>The total years volunteered.</td>
+                <td>This is a computed field based on the volunteer's total years volunteered.</td>
```

Comment:
I'd rework to avoid the passive "based on" formation. I'd change it to: "This computed field calculates the total years that the constituent volunteered."